bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/async/TestAsyncAdmin.java

```diff
@@ -136,18 +139,38 @@ public void testCreateTable_harness() throws Exception {
       // assertEquals(10, patTableDescriptors.get(0).getColumnFamilies()[0].getTimeToLive());
 
       // test listTableDescriptors by List<TableName>
+      try {
+        asyncAdmin.listTableDescriptors(null).get();
+        fail("listTableDescriptors should throw an exception");
+      } catch (Exception e) {
+        Assert.assertTrue(e instanceof NullPointerException);
```

Comment:
Please be consistent in your use of static imports

Reply:
Addressed